.github/images/entrypoint.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+
+echo -e "\n\n--------- Starting the configuration of rskj ---------\n\n"
+cd /usr/src/
+git clone https://github.com/rsksmart/rskj.git rskj
+cd rskj && git checkout $RSKJ_BRANCH
+chmod +x ./configure.sh && chmod +x gradlew
+./configure.sh
+./gradlew --no-daemon clean build -x test
+
+echo -e  "\n\n--------- Starting the configuration of powpeg ---------\n\n"
+cd /usr/src/
+git clone https://github.com/rsksmart/powpeg-node.git powpeg
+cp configure_gradle_federator.sh powpeg
+cd powpeg && git checkout $FEDERATOR_BRANCH
+chmod +x ./configure.sh && chmod +x gradlew
+./configure_gradle_federator.sh
+./configure.sh
+./gradlew  --info --no-daemon clean build -x test
+
+echo -e "\n\n--------- Starting the configuration of RIT ---------\n\n"
+cd /usr/src/
+git clone https://github.com/rsksmart/rootstock-integration-tests.git rit
+mv configure_rit_locally.sh rit
+mv regtest.js rit/config/regtest.js
+mv /usr/src/logbacks/* /usr/src/rit/logbacks/
+cd rit
+git checkout $RIT_BRANCH
+chmod +x ./configure.sh
+./configure.sh
+./configure_rit_locally.sh "$FED_VERSION"
+export LOG_LEVEL=$RIT_LOG_LEVEL
+
+echo -e "\n\n--------- Executing Rootstock Integration Tests ---------\n\n"
+npm install -y
+npm run test-fail-fast
+STATUS=$?
+
+echo -e "\n\n--------- RIT Tests Result ---------\n\n"
+if [ $STATUS -ne 0 ]; then
+  MESSAGE="Rootstock Integration Tests Status: FAILED"
+else
+  MESSAGE="Rootstock Integration Tests Status: PASSED"
+fi
+echo -e "$MESSAGE"
+
+echo "status=$STATUS" >> $env:GITHUB_OUTPUT
+echo "message=$MESSAGE" >> $env:GITHUB_OUTPUT
```

Comment:
```suggestion
echo "status=$STATUS" >> $GITHUB_ENV
echo "message=$MESSAGE" >> $GITHUB_ENV
```

Reply:
You should be able to set the output so that it can be picked up in a following step on the workflow, but you need to reference the GITHUB_OUTPUT variable as ${GITHUB_OUTPUT}
The curly braces shouldn't be mandatory, but it's always better to use them in bash.

Reply:
Ok, will test this then. Let's see how it goes. 🙏